github/issues_labels.go

```diff
@@ -12,13 +12,14 @@ import (
 
 // Label represents a GitHub label on an Issue
 type Label struct {
+	ID    *int    `json:"id,omitempty"`
 	URL   *string `json:"url,omitempty"`
 	Name  *string `json:"name,omitempty"`
 	Color *string `json:"color,omitempty"`
 }
 
 func (l Label) String() string {
-	return fmt.Sprint(*l.Name)
+	return *l.Name
```

Comment:
Actually, there's a test locking in this behavior:

                {Label{Name: String("l")}, "l"},

So I'm going to just remove this from this CL.